packages/parchment/lib/src/document/line.dart

```diff
@@ -26,6 +26,12 @@ final class LineNode extends ContainerNode<LeafNode> with StyledNode {
     return false;
   }
 
+  EmbedNode get embedNode {
```

Comment:
Having a getter that may throw an assertion error seems dangerous here

Reply:
True, I removed it and I'll work without it.